### PR TITLE
Ta15775 backbone level improvements

### PR DESCRIPTION
**Pull request by LeopoldoFu**
@ebsco/eac-software-engineers Fun times.

### CODE REVIEW

File: backbone.js [hunk 1]
```diff
@@ -400,29 +399,16 @@
...
       for (var attr in diff) {
-        if (_.isEqual(old[attr], (val = diff[attr]))) continue;
+        val = diff[attr];
+        if (this.attributes[attr] === val && !(attr in this.changed)) continue;
         (changed || (changed = {}))[attr] = val;
       }
       return changed;
     },
 
-    // Get the previous value of an attribute, recorded at the time the last
```
Comment (Rogibb111) [inline backbone.js:413]: good riddance

File: backbone.js [hunk 2]
```diff
@@ -400,29 +399,16 @@
     // You can also pass an attributes object to diff against the model,
     // determining if there *would be* a change.
     changedAttributes: function(diff) {
-      if (!diff) return this.hasChanged() ? _.clone(this.changed) : false;
```
Comment (Rogibb111) [inline backbone.js:403]: no more clone!

File: backbone.js [hunk 3]
```diff
@@ -341,8 +340,8 @@
       // For each `set` attribute, update or delete the current value.
       for (attr in attrs) {
         val = attrs[attr];
-        if (!_.isEqual(current[attr], val)) changes.push(attr);
```
Comment (Rogibb111) [inline backbone.js:344]: boo underscore functions

**Comment by Rogibb111** [comment]
Looks good to me.

File: backbone.js (hunk 2 above)
Comment (LeopoldoFu) [inline backbone.js:403]: image

**Comment by leachryanb** [comment]
You gonna submit a PR back to jashkenas?

**Comment by LeopoldoFu** [comment]
@leachryanb no, We are truly forking backbone here;  we've changed how backbone handles diffing model changes from deep comparison to simple === comparison and also removed the functionality to get the previous model attributes at any time.

**Comment by leachryanb** [comment]
Huh, wonder if we should change the name to eis-backbone or something.

On Wed, May 20, 2015 at 2:59 PM, LeopoldoFu notifications@github.com
wrote:

> @leachryanb https://github.com/leachryanb no, We are truly forking
> backbone here; we've changed how backbone handles diffing model changes
> from deep comparison to simple === comparison and also removed the
> functionality to get the previous model attributes at any time.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/ebsco/backbone/pull/2#issuecomment-103996079.

## 

-rbl

**Pull request merged**